DocumentDataAPI/DocumentDataAPI/Data/Repositories/Helpers/DapperSqlHelper.cs

```diff
@@ -53,6 +55,15 @@ public string GetBatchInsertParameters<T>(T[] models, out Dictionary<string, dyn
         return stringBuilder.ToString();
     }
 
+    /// <inheritdoc/>
+    public string GetParameterString(QueryParameter param)
+    {
+        string paramString = $"{param.Key} {param.ComparisonOperator} ";
+        if (param.Value is IEnumerable) paramString += $"any(@{param.Key})";
```

Comment:
Ift. det vi snakkede om med hvad der er mest efficient - hvis der kun er én værdi så kan vi bare bruge `=`
```suggestion
        if ((param.Value as IEnumerable)?.Count() > 1) paramString += $"any(@{param.Key})";
```

Reply:
Jeg testede med explain, og optimizeren håndterer det fint ved selv at ændre det til en `=`  så costen er det samme. Vil du stadig gerne have det her? Det er jo i princippet bare dyrere for vores API

Reply:
Så er det fint!